test/Feature/InsertSpec.hs

```diff
@@ -145,7 +145,7 @@ spec = do
           `shouldRespondWith` ResponseMatcher {
             matchBody    = Just inserted
           , matchStatus  = 201
-          , matchHeaders = ["Location" <:> [str|/json?data=eq.{"foo":"bar"}|]]
+          , matchHeaders = ["Location" <:> "/json?data=eq.%7B%22foo%22%3A%22bar%22%7D"]
```

Comment:
You mentioned somewhwere that there is a problem getting back the record with the value encoded, does it work now as it is?